### DISCUSSION TITLE
Command pull from git error  `assert includeFile`

### DISCUSSION THREAD

**Discussion by liuyang2006** [nextflow-io/nextflow#2313] [Q&A] [answered]
Hi everyone, I encountered nextflow pull or run error when I specify my Github repository. Can you help me to figure out the problem? Thank you!

Below is the pull command error.
```
nextflow pull https://github.com/liuyangzzu/nanome

Checking https://github.com/liuyangzzu/nanome ...
assert includeFile
       |
       [:]
```

**Comment by pditommaso** [comment, marked as answer]
Solved by https://github.com/nextflow-io/nextflow/issues/2314
